packages/paste-core/components/chat-log/src/ChatBubble.tsx

```diff
@@ -41,6 +41,7 @@ export const ChatBubble = React.forwardRef<HTMLDivElement, ChatBubbleProps>(
         element={element}
         ref={ref}
         variant={variant}
+        whiteSpace="pre-wrap"
```

Comment:
Solves the line breaks in user generated strings

Reply:
Oh, neat